cmd/defaults.yaml

```diff
@@ -35,15 +35,14 @@ AdminUser:
     Key: ""
 
 Projections:
-  Config:
-    RequeueEvery: 10s
-    RetryFailedAfter: 1s
-    MaxFailureCount: 5
-    BulkLimit: 200
-    MaxIterators: 1
-    Customizations:
-      projects:
-        BulkLimit: 2000
+  RequeueEvery: 10s
+  RetryFailedAfter: 1s
+  MaxFailureCount: 5
+  BulkLimit: 200
+  MaxIterators: 1
+  Customizations:
+    projects:
```

Comment:
Title

Reply:
`project` is written lowercase because all the customisations are looked up like that
![image](https://user-images.githubusercontent.com/9405495/160343075-1d86c16c-78fe-4b6b-a2da-8fb7fba53a78.png)